include/fmt/format-inl.h

```diff
@@ -698,7 +698,7 @@ FMT_API bool grisu_format(Double value, buffer<char>& buf, int precision,
       buf.push_back('0');
     } else {
       exp = -precision;
-      buf.resize(precision);
+      buf.resize(static_cast<size_t>(precision));
```

Comment:
Please use `to_unsigned` instead of `static_cast`:

```c++
buf.resize(to_unsigned(precision));
```

Reply:
Fixed